test/hotspot/jtreg/compiler/c2/TestEliminateRedundantConversionSequences.java

```diff
@@ -23,13 +23,19 @@
 
 /*
  * @test
- * @bug 8359603
+ * @bug 8359603 8369646
  * @summary Redundant ConvX2Y->ConvY2X->ConvX2Y sequences should be
  *          simplified to a single ConvX2Y operation when applicable
  *          VerifyIterativeGVN checks that this optimization was applied
  * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions
  *      -XX:CompileCommand=compileonly,compiler.c2.TestEliminateRedundantConversionSequences::test*
- *      -XX:-TieredCompilation -Xbatch -XX:VerifyIterativeGVN=1110 compiler.c2.TestEliminateRedundantConversionSequences
+ *      -XX:-TieredCompilation -Xbatch -XX:VerifyIterativeGVN=1110
```

Comment:
You could either add a separate run with `-XX:+StressIGVN` without a fixed seed or just add `-XX:+StressIGVN` here. I guess the latter is good enough.